docker-init-script/src/test/java/eu/xenit/docker/alfresco/InitScriptMainTest.java

```diff
@@ -212,4 +224,43 @@ public void testSetCustomLoggerLevels() throws IOException {
         checkLoggers("testSetCustomLoggerLevels", env);
     }
 
+    @Test
+    public void testSetCustomDirKeystore() throws IOException {
+        Map<String, String> env = getDefaultEnvironment();
+        String customKeystoreDir = "/ssl/keystores";
+        env.put("DIR_KEYSTORE", customKeystoreDir);
+        InitScriptMain main = new InitScriptMain(env, new HashMap<String, String>(), new HashMap<String, String>());
+        main.process();
+        assertTrue("JAVA_OPTS contains values for custom key- and truststore locations.",
+                main.getJavaOptions().containsAll(
+                        Arrays.asList("-DTOMCAT_SSL_KEYSTORE=".concat(customKeystoreDir).concat("/ssl.keystore"),
+                                "-DTOMCAT_SSL_TRUSTSTORE=".concat(customKeystoreDir).concat("/ssl.truststore"))));
+    }
+
+    @Test
+    public void testSetCustomKeystores() throws IOException {
+        Map<String, String> env = getDefaultEnvironment();
+        String methodname = "testSetCustomKeystores";
+        String testKeystoreLocation = methodname.concat("/");
+        List<String> keystorePasswordJavaOpts = new ArrayList<>();
+        keystorePasswordJavaOpts.add("-DTOMCAT_SSL_KEYSTORE_PASSWORD=".concat(getKeystorePasswordFromProps(testKeystoreLocation.concat("/ssl-keystore-passwords.properties"))));
+        keystorePasswordJavaOpts.add("-DTOMCAT_SSL_TRUSTSTORE_PASSWORD=".concat(getKeystorePasswordFromProps(testKeystoreLocation.concat("/ssl-truststore-passwords.properties"))));
```

Comment:
Instead of reading it from the file here, can you please hardcode the expected truststore password?